rotkehlchen/api/rest.py

```diff
@@ -4984,6 +4984,25 @@ def add_tokens_to_spam(self, tokens: list[EvmToken]) -> Response:
                 )
                 AssetResolver.clean_memory_cache(token.identifier)
 
+                for balances in self.rotkehlchen.chains_aggregator.balances.get(
+                    chain=(blockchain := token.chain_id.to_blockchain()),
+                ).values():
+                    # those can only be dicts because we mark as spam only evm tokens
+                    in_assets = balances.assets.pop(token, None)  # type: ignore
+                    in_liabilities = balances.liabilities.pop(token, None)  # type: ignore
+
+                    if in_assets is not None or in_liabilities is not None:
+                        # don't flush the cache of query_{blockchain}_balances since
+                        # we don't need to clear it and prevents us from querying the
```

Comment:
Also here it's unclear: `and prevents us from querying the blockchain again` ... I don't understand what you are trying to say. Please write the comment in discord in English and try to explain to me what you want to say. Then we can just have it put here without going through a PR review cycle.